### PR TITLE
Fix texture path

### PR DESCRIPTION
**Pull request by ochounos**
Fix issue #182. Solve error with relative path textures. The config var texture_dir is not used to calculate the right path.

### CODE REVIEW

File: io_export_cryblend/export.py [hunk 1]
```diff
@@ -195,9 +195,23 @@ def __export_library_images(self, parent_element):
             self.__convert_images_to_dds(images)
 
     def __export_library_image(self, image):
+        relative_path = bpy.path.abspath(self.__config.filepath)
+        last_slash = relative_path.rfind("\\")
+        if last_slash != -1:
+            relative_path = relative_path[:last_slash]
+
```
Comment (dwhite4) [inline io_export_cryblend/export.py]: Perhaps a simpler method would be:
```python
relative_path = os.path.dirname(bpy.path.abspath(self.__config.filepath))
```

This has the effect of cutting off the filename from the path.

Comment (ochounos) [inline io_export_cryblend/export.py]: Modified already the three path calculation. Seems to work for me, please review it again.

**Comment by dwhite4** [comment]
Awesome, looks good.  Ready for merge.

**Comment by dwhite4** [comment]
@ochounos, @feedbackex,
#187 #188 seem to be on the same topic, you guys may want to discuss which should be implemented.

**Comment by ochounos** [comment]
Improved code on #188 pull request.

**Pull request closed without merging**